docs/software-development-kits/java/middleware.mdx

```diff
@@ -0,0 +1,133 @@
+---
+sidebar_label: Spring Middleware
+title: Spring Middleware
+description: Aserto Middleware - Java - Installing and using Spring Middleware
+---
+
+The `AsertoFilter` is a Java library that provides a Spring Security filter that enables the use of Aserto's authorization service in projects that use Spring.
+
+## Requirements
+- Java 8  or newer
+- Spring Boot 2.7.x
+- Spring Security 5.7.x
+
+### Installation
+Add the fallowing dependency to your `pom.xml` file:
+
+```xml
+<dependency>
+    <groupId>com.aserto</groupId>
+    <artifactId>aserto-spring</artifactId>
+    <version>0.0.2</version>
+</dependency>
+```
+
+The latest version is available on [Maven Central](https://search.maven.org/artifact/com.aserto/aserto-spring).
+
+And them execute:
+```bash
+mvn clean package
+```
+
+In order for Spring to discovery the filter, we need to add the `@ComponentScan` annotation to the main class of the application.
+```java
+@ComponentScan("com.aserto")
+```
+
+## Configuration
+
+The authorization filter accepts the fallowing configuration parameters:
+
+| Parameter name | Default Value | Description |
+| -------------- | ------------- | ----------- |
+| aserto.authorization.enabled | true | Enable/disable the authorization filter |
+| aserto.authorization.host | localhost | The hostname of the authorizer service |
+| aserto.authorization.port | 8282 | The port of the authorizer service |
+| aserto.authorizer.insecure | false | Use insecure connection to the authorizer service |
+| aserto.authorizer.cert.path | N/A | The path to the authorizer service certificate, usefull when using local topaz with self signed certificates |
+| aserto.authorizer.apiKey | N/A | The API key to use when connecting to the authorizer service |
+| aserto.tenantId | N/A | The tenant ID to use when connecting to the authorizer service |
+| aserto.authorizer.policyName | N/A | The policy name used by the authorizer |
+| aserto.authorizer.policyLabel | N/A | The policy label used by the authorizer |
+| aserto.authorizer.decision | N/A | The decision used by the authorizer |
+| logging.level.com.aserto | N/A | The log level for the Aserto Middleware |
```

Comment:
Is there really no default value for the log level? Does that mean logging is disabled by default?

Reply:
This is a mistake, the default is INFO